### PR TITLE
[ONNX] concat （axis= 0）

### PR DESCRIPTION
**Pull request by Deliadd**

### CODE REVIEW

**Comment by Deliadd** [comment]
Jenkins passed. ID is 1427.

File: pyzoo/zoo/pipeline/api/onnx/mapper/concat.py [hunk 1]
```diff
@@ -23,12 +23,8 @@ def __init__(self, node, _params, _all_tensors):
         super(ConcatMapper, self).__init__(node, _params, _all_tensors)
 
     def _to_tensor(self):
-        axis = 0
+        axis = -1
```
Comment (zhichao-li) [inline pyzoo/zoo/pipeline/api/onnx/mapper/concat.py]: the default axis is 0?

Comment (zhichao-li) [reply pyzoo/zoo/pipeline/api/onnx/mapper/concat.py]: assert 'axis' in self.onnx_attr, "axis is a required attribute"

**Comment by Deliadd** [comment]
Jenkins passed. ID is 1429

**Pull request merged**